frontend/public/components/service.jsx

```diff
@@ -25,7 +25,7 @@ const ServiceIP = ({s}) => {
     return <div key={i}>{clusterIP}</div>;
   });
 
-  return <p>{children}</p>;
+  return <div>{children}</div>;
```

Comment:
You might be able to make this a `<React.Fragment>` now. Can you check if that renders the same?

Reply:
Actually you might be able to simply do

```jsx
return children;
```

Reply:
I just tested with `return children;`, and that behaves correctly.